packages/relay/src/lib/hbarlimiter/index.ts

```diff
@@ -0,0 +1,73 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import dotenv from 'dotenv';
+import findConfig from 'find-config';
+
+export default class HbarLimit {
+    private remainingBudget: number;
+    private total: number;
+    private duration: number;
+    private reset: number;
+
+    constructor() {
```

Comment:
Please update this class to rely of data time pass ins.
I think this would allow for easier testing and clarity.

Then you can add a hbarLimiter.spec.ts for coverage
```suggestion
    constructor(currentDateNow) {
```